pkg/virt-controller/services/template.go

```diff
@@ -59,6 +60,10 @@ const CAP_SYS_NICE = "SYS_NICE"
 // Libvirt needs roughly 10 seconds to start.
 const LibvirtStartupDelay = 10
 
+//This is a perfix for node feature discovery, used in a NodeSelector on the pod
+//to match a VirtualMachineInstance CPU model(Family) to nodes that support this model.
+const NFD_CPU_FAMILY_PREFIX = "feature.node.kubernetes.io/cpu-model-"
```

Comment:
Is it common to do `feature.node.kubernetes.io/cpu-model-mymodel: true` over `feature.node.kubernetes.io/cpu-model: mymodel`?

Reply:
This is how NFD will label the nodes.  The question is logical, If the labelling method will change then we will adapt.

Reply:
:+1: